src/js/components/reviewData/ReviewDataComments.jsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * ReviewDataComments.jsx
+ * Created by Alisa Burdeyny 11/21/16
+ **/
+
+import React from 'react';
+import ReviewDataCommentsDropdown from './ReviewDataCommentsDropdown.jsx'
+import ReviewDataCommentsTextfield from './ReviewDataCommentsTextfield.jsx'
+
+import * as ReviewHelper from '../../helpers/reviewHelper.js';
+
+export default class ReviewDataComments extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            currentFile: "A",
+            fileComments: props.comments,
+            saveState: ""
+        };
+    }
+
+    changeFile(newFile) {
+        var fileComment = document.getElementById("submission-review-comments").value.trim();
+        var tempComments = this.state.fileComments;
+        tempComments[this.state.currentFile] = fileComment;
+
+        this.setState({
+            fileComments: tempComments,
+            currentFile: newFile,
+        });
+    }
+
+    saveComments() {
+        this.setState({saveState: "Saving"});
+        var fileComment = document.getElementById("submission-review-comments").value.trim();
```

Comment:
Same issues as above; also, this is duplicated code. Could you consolidate the logic of saving the comments to state in a single, reusable function?